### PR TITLE
Halium chroot script

### PR DESCRIPTION
**Pull request by ZeroPointEnergy**
This is a simple script to directly log into a halium chroot via adb/recovery for debugging purposes. It makes it a lot less painful to read the logs or change stuff if systemd is stuck and no ssh is available yet.

### CODE REVIEW

**Comment by z3ntu** [comment]
Great idea.

File: halium-chroot [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+CHROOT='TERM=linux HOME=/root PATH=/sbin:/bin:/usr/sbin:/usr/bin:$PATH chroot /target'
+LINES=$(tput lines)
+COLUMNS=$(tput cols)
+
+if [ $# -eq 0 ]
+then # Interactive shell
+	if [ $(which tmux) ]
+	then # with tmux installed
+		tmux new-session -s halium-adb -d 'adb shell'
```
Comment (bhush9) [inline halium-chroot]: This won't probably give me shell if I am already in the tmux session :-P

Comment (ZeroPointEnergy) [reply halium-chroot]: Well that is true :-) but without a proper terminal multiplexer it is a pain to get an interactive shell where you can inject the commands to start the chroot. Maybe I can detect an already running session and work around that.

Comment (bhush9) [reply halium-chroot]: yeah, would be lovely.. :)

**Comment by bhush9** [comment]
Thanks, will merge it :)

**Pull request merged**